### PR TITLE
Feature/{chimereucheya}/About

### PR DESCRIPTION
**Pull request by Merdoth**
#### What does this PR do?
-Application About Page

#### Description of Task to be completed?
- Shows the application about page which gives an in-depth introduction about what lifeAid, who works in lifeAid and a couple of other basic information the public needs to know.

#### How should this be manually tested?
- run the application using `npm run dev` visit `http://localhost:3000/about`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- story type: Feature
- story title: About Page

#### Screenshots (if appropriate)
N/A

#### Question
N/A

### CODE REVIEW

File: components/About/about.scss [hunk 1]
```diff
@@ -0,0 +1,383 @@
+.header {
```
Comment (Way2nnadi) [inline components/About/about.scss:1]: look through this file, you still have minor formatting issues

File: components/About/AboutPage.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                    <br />
+                    {' '}
+                    {INTRO_TEXT2}
+                </p>
+                <span>
+                    <p>
+                        {INTRO_HEADER_TEXT}
+                        <br />
+                        <br />
+                        <br />
```
Comment (Way2nnadi) [inline components/About/AboutPage.js:51]: why are you abusing the `br` tag element, why don't you just use css to create the needed `margin`

**Review by Way2nnadi — APPROVED** [review]
Minors changes but looks good overall

**Review by codemutatio — APPROVED** [review]

**Review by iammiracle — APPROVED** [review]
I think you can go ahead with the merge

File: components/About/about.scss (hunk 1 above)
Comment (Merdoth) [reply components/About/about.scss:1]: On it

Comment (Merdoth) [reply components/About/about.scss:1]: Resolved

File: components/About/AboutPage.js (hunk 2 above)
Comment (Merdoth) [reply components/About/AboutPage.js:51]: The css margin wasn't giving me the exact precise space I needed

**Pull request merged**